### PR TITLE
Tenant manager / identification

### PR DESCRIPTION
**Pull request by jimleeder123**
This is for the set up of the tenant manager and identifying which tenant you are on using the `ref` variable in the URL query string.

If the var is set to a valid tenant enum then that enum is saved to the session. It falls back to the WFG enum by default.

### CODE REVIEW

File: app/Enums/TenantEnum.php
Comment (gbuckingham89) [inline app/Enums/TenantEnum.php:1]: Can we we drop the `Enum` prefix from the class name? (https://dev-handbook.yammayap.com/php/best-practices/)

File: app/Services/TenantManager/TenantManager.php
Comment (gbuckingham89) [inline app/Services/TenantManager/TenantManager.php:1]: No much of an issue here as it's only a relatively small class, but grumpy reviewer would question the order of the method definitions - we'd usually go alphabetical / or public then private, sub-sorted by alphabetical too - something to consider for future classes.

File: resources/css/app.css
Comment (gbuckingham89) [inline resources/css/app.css:1]: Grumpy reviewer would say this doesn't relate to this PR, but doesn't mater too much.

File: resources/views/skeleton.blade.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="csrf-token" content="{{ csrf_token() }}">
+    <meta name="robots" content=”noindex,nofollow”>
```
Comment (gbuckingham89) [inline resources/views/skeleton.blade.php:7]: Can we remove this line - the robots tag is controlled can be / should be set by the SEO package.

File: resources/views/skeleton.blade.php [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    <link href="https://fonts.googleapis.com/css?family=Lato:400,700" rel="stylesheet">
```
Comment (gbuckingham89) [inline resources/views/skeleton.blade.php:8]: We don't know we'll be using this - so can we get rid for now please!

_So we don't forget exists and end up loading unnesscary things!_

File: resources/views/skeleton.blade.php
Comment (gbuckingham89) [inline resources/views/skeleton.blade.php:1]: Again, grumpy reviewer would question why it's in this PR, but happy to leave it for now if we tidy it up!

File: routes/web.php [hunk 3]
```diff
@@ -1,12 +1,12 @@
 <?php
 
+use App\Http\Controllers\DashboardController;
```
Comment (gbuckingham89) [inline routes/web.php:3]: Unused import! 

Remind me and I'll show you a PHPStorm setting that automatically tidies these up :)

**Review by gbuckingham89 — CHANGES_REQUESTED** [review]
Overall nice job, especially using the container and mocking for the first time 🚀! Just a few small bits to tidy up.

Also, I know the company logos are only temporary but can we give some thought about where we store them - maybe moving the `tenants` directory inside a new `public/img` directory would be a good idea, so eventually all images live in one place?

_I know there is a lot to take in with the container and mocking - but the more we do it, the more it will become clearer! But do say if something doesn't make sense and I'll try to explain better._

File: app/Enums/TenantEnum.php
Comment (jimleeder123) [reply app/Enums/TenantEnum.php:1]: Done

File: app/Services/TenantManager/TenantManager.php
Comment (jimleeder123) [reply app/Services/TenantManager/TenantManager.php:1]: Have done this so it's public then private and sorted alphabetically.

File: resources/css/app.css
Comment (jimleeder123) [reply resources/css/app.css:1]: The file has been cleared back to it's original state, this was just something to get it working while I was doing the temp test view for visual purposes

File: resources/views/skeleton.blade.php (hunk 1 above)
Comment (jimleeder123) [reply resources/views/skeleton.blade.php:7]: Removed

File: resources/views/skeleton.blade.php (hunk 2 above)
Comment (jimleeder123) [reply resources/views/skeleton.blade.php:8]: Removed

File: resources/views/skeleton.blade.php
Comment (jimleeder123) [reply resources/views/skeleton.blade.php:1]: It was to get the temp test view working. It is mostly copied from MPMR and have tidied it up

File: routes/web.php (hunk 3 above)
Comment (jimleeder123) [reply routes/web.php:3]: Removed

**Review by gbuckingham89 — APPROVED** [review]
Looks good - nice work! Let's get the first "real" PR for widgets merged 🥳🍾

**Pull request merged**